### DISCUSSION TITLE
Unable to add device

### DISCUSSION THREAD

**Discussion by themabrennan** [milo526/homebridge-tuya-web#360] [General]
I cannot add two new lights, 'Lumary Smart Ceiling Light A1', into the plugin. I was able to run the python script, and it did NOT locate the lights, so I reached out to support@tuya.com. Their response is below: 

"Dear sir/madam,

I hope you have a nice day today.

Since Tuya does not currently support Home Assistant product development, products that previously supported Home Assistant will continue to support it. Products that do not support Home Assistant currently only support the development of individual developers. You can refer to: https://www.home-assistant.io/components/tuya/ Tuya will no longer support the development of new products.

In addition, if you have questions about the development of the IOT platform, it is recommended that you directly log in to the following link to ask questions: https://service.console.tuya.com/.

Thank you for your support and understanding."

I referred to their instructions regarding Home Assistant, creating an account, project, etc., and added the lights I could NOT find via the python script. I decided to set up home assistant to see if it would recognize those two lights, and sure enough, it was able to locate and control." 

After setting up home assistant, I went back to the plugin to see if the lights would then be added, and they are still not. Do you have any suggestions?

**Comment by milo526** [comment]
If the python script can't find them; this plugin can't use them.

Please use Tuya's new first-party plugin instead, that should work for you.
https://github.com/tuya/tuya-homebridge
